modules/RTC/RTCUtils.js

```diff
@@ -847,6 +847,15 @@ var RTCUtils = {
 
                 //AdapterJS.WebRTCPlugin.setLogLevel(
                 //    AdapterJS.WebRTCPlugin.PLUGIN_LOG_LEVELS.VERBOSE);
+
+                AdapterJS.WebRTCPlugin.isPluginInstalled(
```

Comment:
Does this still show the UI for installing temasys? I'm not sure if that's separate from this flow but hopefully it is.